src/System.Management.Automation/namespaces/ProviderBase.cs

```diff
@@ -1862,6 +1862,27 @@ public void WriteInformation(object messageData, string[] tags)
 
                 result.AddOrSetProperty("PSChildName", childName);
                 providerBaseTracer.WriteLine("Attaching {0} = {1}", "PSChildName", childName);
+#if UNIX
+
+                if (ExperimentalFeature.IsEnabled("PSUnixFileStat"))
+                {
+                    // Add a commonstat structure to file system objects
+                    if (ProviderInfo.ImplementingType == typeof(Microsoft.PowerShell.Commands.FileSystemProvider))
```

Comment:
Could squash this into the previous condition with an `&&`

Reply:
done